index.html

```diff
@@ -97,10 +97,24 @@
 (for example, with strong privacy protections).
 
 The web should also support human rights, dignity and personal agency.
-We need to put internationally recognized human rights at the core of the web platform. [[?UDHR]]
+We need to put internationally recognized human rights at the core of the web platform [[?UDHR]].
 And we need to promote ethical thinking across the web industry
 to reinforce this approach.
 
+The principles in this document are deliberately unordered,
+and many are interconnected with each other.
+While all of the principles together aim to provide pillars that collectively support
+a web that is beneficial for society, there are cases where upholding one principle
+may diminish the efficacy of another principle.
+Thus in applying these principles, there are benefits and tradeoffs
+that may need to be carefully balanced. When faced with principles
+which appear to be in conflict with one another,
+it is important to consider the context in which a particular technology
+is being applied, the expected audience(s) for the technology,
+who the technology benefits and who it may disadvantage,
+and any power dynamics involved.
```

Comment:
This seems very reminiscent of the priority of constituencies. Referencing it here does create somewhat of a circular dependency problem, though.

Curiously, the priority of constituencies seems to be both a concrete implementation of some of these principles in the platform _and_ (as you describe here) a sort of meta principle that sits as an arbiter between said principles. I feel like @mnot addresses this tension well in [RFC 8890](https://www.rfc-editor.org/rfc/rfc8890) by tying the end-user focus to IETF's mission. Maybe we need something similar for W3C, to your point in #37 and @torgo's comment in #58?

In the meantime, I'd suggest either merging as is (and making a note of that tension in an issue) or replacing most of the selected text by a reference to the priority of constituencies (and filing an issue to come back and deal with the circularity of it all).

Reply:
Thanks @tobie! I actually was going to add the priority of constituencies (which I don't think is a circular dependency, as that's in the Design Principles, which are already circularly referenced by the EWP) but then I realised it is mentioned immediately afterwards in 2.2. But maybe it is a good point to belabour. I don't think it's *instead of* though, as you could conceivably end up with competing principles applying to the same constituent, as well as principles that come into competition when applied to different constituents.

Reply:
> I don't think it's _instead_ of though, as you could conceivably end up with competing principles applying to the same constituent,[…]

Ha! That’s a great point I hadn’t thought about. That sounds like the right way to go about this!